proof/infoflow/ARM/Example_Valid_State.thy

```diff
@@ -1635,14 +1621,18 @@ lemma valid_arch_caps_s0[simp]:
   apply (drule s0_caps_of_state)+
   by auto
 
+\<comment> \<open>FIXME MK: move, or kernel_base could be changed to be an abbreviation of pptrBase\<close>
+lemma kernel_base_aligned_pageBits:
+  "is_aligned kernel_base pageBits"
+  by (simp add: is_aligned_def kernel_base_def pageBits_def)
```

Comment:
@corlewis, can you check how often we mention `kernel_base`? It'd be good to get rid of it completely and replace globally by `pptrBase`. It's from ages back when terminology hadn't settled yet and ASpec/Haskell/C all had different names for the same thing.

Reply:
It's used enough that it might be a bit too much to replace it as part of this PR. Across ARM, ARM_HYP and X64 there's 39 instances of it in `spec` and 391 in `proof`.

Reply:
I hit the same issue when I was doing kernel constant renames to match C (see "arm+arm-hyp: kernelBase and physMappingOffset renames" commit on l4v). `kernel_base` should definitely go, but I agree to leave it out of this PR.

- [ ] create an issue for kernel_base rename to pptrBase